README.md

```diff
@@ -140,13 +140,25 @@ In order to use it, you need to create the blacklist table in a migration:
 def change
   create_table :jwt_blacklist do |t|
     t.string :jti, null: false
+    t.datetime :revoked_at, default: Time.now
+    t.datetime :expiration_time
```

Comment:
@waiting-for-dev , sqlite requires that a not null datetime have a default value. You thoughts on what that should be?